testutil/http.go

```diff
@@ -37,6 +37,7 @@ func HTTPServerAndRouter(t *testing.T) (*httptest.Server, *httpstub.Router) {
 	r := httpstub.NewRouter(t)
 	r.Method(http.MethodPost).Path("/users").Response(http.StatusCreated, nil)
 	r.Method(http.MethodPost).Path("/help").Response(http.StatusCreated, nil)
+	r.Method(http.MethodPost).Path("/upload").Response(http.StatusCreated, nil)
```

Comment:
@atsushi-ishibashi 

About https://github.com/k1LoW/runn/pull/283#issuecomment-1316396837 ,

I think another test (`TestSetUploadFile`) is failing because of the duplicate `/upload` path.

https://github.com/k1LoW/runn/pull/283/files#diff-d833779323fc4c6d7f044dc910cb45153e44fb79b85e01d39d77d6aa8a6f9b67R53